src/Symfony/Component/Uid/Uuid.php

```diff
@@ -27,6 +27,12 @@ class Uuid extends AbstractUid
     protected const NIL = '00000000-0000-0000-0000-000000000000';
     protected const MAX = 'ffffffff-ffff-ffff-ffff-ffffffffffff';
 
+    public const FORMAT_BINARY = 1;
+    public const FORMAT_BASE_32 = 1 << 1;
+    public const FORMAT_BASE_58 = 1 << 2;
+    public const FORMAT_RFC_4122 = 1 << 3;
+    public const FORMAT_ALL = self::FORMAT_BINARY | self::FORMAT_BASE_32 | self::FORMAT_BASE_58 | self::FORMAT_RFC_4122;
```

Comment:
public consts should be grouped together, before protected ones
```suggestion
    public const FORMAT_ALL = -1;
```

Reply:
Oh smart, thanks!